lib/uploadcare/rails/api/rest/conversion_api.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'uploadcare/rails/api/rest/base'
+
+module Uploadcare
+  module Rails
+    module Api
+      module Rest
+        # A class that contains Conversion related methods for Uploadcare REST API
+        class ConversionApi < Base
+          class << self
+            # Converts video files
+            # @see https://uploadcare.com/api-refs/rest-api/v0.5.0/#operation/videoConvert
+            def convert_video(video_params, **options)
+              Uploadcare::VideoConverter.convert(video_params, **options)
+            end
+
+            # Returns a status of video conversion job
+            # @see https://uploadcare.com/api-refs/rest-api/v0.5.0/#operation/videoConvertStatus
+            def get_video_conversion_status(token)
+              Uploadcare::VideoConverter.status(token)
+            end
+
+            # Converts documents
+            # @see https://uploadcare.com/api-refs/rest-api/v0.5.0/#operation/documentConvert
+            def convert_document(document_params, **options)
+              Uploadcare::DocumentConverter.convert(document_params, **options)
+            end
+
+            # Returns a status of video conversion job
+            # @see https://uploadcare.com/api-refs/rest-api/v0.5.0/#operation/documentConvertStatus
+            def get_document_conversion_status(token)
+              Uploadcare::DocumentConverter.status(token)
+            end
+          end
+        end
+      end
+    end
+  end
+end
+
+Uploadcare::ConversionApi = Uploadcare::Rails::Api::Rest::ConversionApi
```

Comment:
Why don't we include this API in the gem?

Reply:
We can move this to the ruby-gem but ruby-gem will be changed much because I will have to change existing ruby wrapper classes.

@dmitry-mukhin, what you think about moving all API classes from Rails-gem to Ruby-gem?
And should I significantly edit ruby-gem this way?

Reply:
this definitely looks like it belongs to ruby gem.
but it can me moved there later, when we'll add support for it in core lib.